UICatalog/Scenarios/AllViewsTester.cs

```diff
@@ -348,49 +348,17 @@ private View CreateClass (Type type)
             view.ColorScheme = Colors.ColorSchemes ["Base"];
         }
 
-        // If the view supports a Text property, set it so we have something to look at
-        if (view.GetType ().GetProperty ("Text") != null)
+        if (view is IDesignable designable)
```

Comment:
YAY

Thank you.

So much nicer. Even if it's "just" in samples, this gives me warm fuzzies. :slightly_smiling_face: